docs/Operators.md

```diff
@@ -27310,6 +27311,69 @@ expect(
 </details>
 
 
+### <a name="RotaryEmbedding"></a><a name="rotaryembedding">**RotaryEmbedding**</a>
+
+  RotaryEmbedding is the implementation of rotary positional embeddings (RoPE) based on the paper https://arxiv.org/pdf/2104.09864.
+  The positions are represented as rotation matrices that are multiplied to query and key
```

Comment:
It is unclear what these lines mean. It seems more to do with how and where the op is used. Is there any connection between query/key and any of the parameters below? What is position_ids? What are cos and sin meant to be?

Reply:
Updated description